docs/modules/reference/pages/how-to-build-with-docker.adoc

```diff
@@ -45,7 +45,7 @@ The URL for the data source in Grafana for the OpenNMS Horizon server is `http:/
 
 == Docker configuration for PDF dashboard reports
 
-OpenNMS Horizon/Meridian includes three templates to create a PDF report from an existing Grafana dashboard. 
+OpenNMS Horizon/Meridian includes three templates to https://docs.opennms.com/horizon/28.0.0/operation/database-reports/database.html#creating-a-pdf-of-a-grafana-dashboard[create a PDF report] from an existing Grafana dashboard. 
```

Comment:
As asked previously - how do we keep this updated with current OpenNMS release versions?

Reply:
I should be able to link internally with the component. Will update both links.